slasher/detection/proposer_slash.go

```diff
@@ -0,0 +1,50 @@
+// Package slasher implements slashing validation
+// and proof creation.
+package slasher
+
+import (
+	"errors"
+
+	"github.com/prysmaticlabs/go-bitfield"
+	"github.com/prysmaticlabs/prysm/shared/sliceutil"
+)
+
+var epochProposalBitlist map[uint64]bitfield.Bitlist
+var currentEpoch, weakSubjectivityPeriod uint64
+var epochs []uint64
+
+func init() {
+	epochProposalBitlist = make(map[uint64]bitfield.Bitlist)
+	weakSubjectivityPeriod = uint64(54000)
```

Comment:
this should be from our config file not hardcoded over here

Reply:
fixed, thanks